vumi/transports/smpp/smpp_transport.py

```diff
@@ -298,6 +298,15 @@ def _reject_for_invalid_address(self, message, field):
         return self.publish_nack(
             message['message_id'], u'Invalid %s: %s' % (field, message[field]))
 
+    def on_connection(self):
+        return self.publish_status_unbound()
+
+    def publish_status_unbound(self):
+        return self.publish_status('smpp', 'major', reasons=[{
+            'type': 'unbound',
+            'message': 'Connected but not bound',
+        }])
```

Comment:
I think what I noted here: https://github.com/praekelt/vumi/pull/984#discussion_r42141800 , applies here too. We don't really have any freely available information besides the config information.